src/components/ChatView.vue

```diff
@@ -108,6 +108,8 @@ export default {
 	mounted() {
 		// Postpone render of NewMessageForm until application is mounted
 		this.containerId = this.$store.getters.getMainContainerSelector()
+
+		this.$store.dispatch('getTranslationConfiguration')
```

Comment:
This means we load the configuration when switching chat to call and back?
Anyway as mentioned in the other comment we have a capability with the value that should be used